_pages/plugins/gut-analysis-toolbox.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Gut Analysis Toolbox
+description: Gut Analysis Toolbox or GAT allows the semi-automated analysis of the cells within the enteric nervous system of the gastrointestinal tract in **2D**. GAT enables quantification of enteric neurons and their subtypes in gut wholemounts. It runs in FIJI, a popular image analysis software in microscopy and uses deep learning models to segment cells of interest. 
```

Comment:
Please spell Fiji with mixed case, not all caps. I know, it's an acronym, but "Fiji" is how the project writes it.